sbol3/document.py

```
@@ -433,9 +432,12 @@ def join_lines(self, lines: List[Union[bytes, str]]) -> Union[bytes, str]:
         if lines_type is bytes:
             # rdflib 5
             return b'\n'.join(lines) + b'\n'
+        # if lines_type is str
         elif lines_type is str:
             # rdflib 6
             return '\n'.join(lines) + '\n'
+        else:
+            return ''
```

Instead of returning an empty string I think this should be an error and we should raise an exception. Maybe a `ValueError` exception could be raised because `lines` is neither a `bytes` or a `str`.

---

Okay. I've replaced the return empty string with  `raise ValueError('Lines must be either bytes or str')`